[2024-09-15T07:28:25.754+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: save_city_info_weather_v01.save_city_info_weather manual__2024-09-15T07:28:24.794509+00:00 [queued]>
[2024-09-15T07:28:25.759+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: save_city_info_weather_v01.save_city_info_weather manual__2024-09-15T07:28:24.794509+00:00 [queued]>
[2024-09-15T07:28:25.759+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-09-15T07:28:25.759+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2024-09-15T07:28:25.759+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-09-15T07:28:25.766+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): save_city_info_weather> on 2024-09-15 07:28:24.794509+00:00
[2024-09-15T07:28:25.770+0000] {standard_task_runner.py:55} INFO - Started process 3605 to run task
[2024-09-15T07:28:25.774+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'save_city_info_weather_v01', 'save_city_info_weather', 'manual__2024-09-15T07:28:24.794509+00:00', '--job-id', '327', '--raw', '--subdir', 'DAGS_FOLDER/saver_info.py', '--cfg-path', '/tmp/tmpadyeswbz']
[2024-09-15T07:28:25.774+0000] {standard_task_runner.py:83} INFO - Job 327: Subtask save_city_info_weather
[2024-09-15T07:28:25.840+0000] {task_command.py:389} INFO - Running <TaskInstance: save_city_info_weather_v01.save_city_info_weather manual__2024-09-15T07:28:24.794509+00:00 [running]> on host 0c5fe11d0c9c
[2024-09-15T07:28:25.932+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=save_city_info_weather_v01
AIRFLOW_CTX_TASK_ID=save_city_info_weather
AIRFLOW_CTX_EXECUTION_DATE=2024-09-15T07:28:24.794509+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-09-15T07:28:24.794509+00:00
[2024-09-15T07:28:25.933+0000] {sql.py:253} INFO - Executing: 
            create table if not exists weathers (
                id serial PRIMARY KEY ,
                city_name VARCHAR,
                max_temp VARCHAR,
                min_temp VARCHAR,
                wind VARCHAR,
                descript VARCHAR
            )
        
[2024-09-15T07:28:25.942+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-15T07:28:26.080+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-15T07:28:26.087+0000] {sql.py:364} INFO - Running statement: 
            create table if not exists weathers (
                id serial PRIMARY KEY ,
                city_name VARCHAR,
                max_temp VARCHAR,
                min_temp VARCHAR,
                wind VARCHAR,
                descript VARCHAR
            )
        , parameters: None
[2024-09-15T07:28:26.094+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=save_city_info_weather_v01, task_id=save_city_info_weather, execution_date=20240915T072824, start_date=20240915T072825, end_date=20240915T072826
[2024-09-15T07:28:26.117+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-09-15T07:28:26.134+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
